samples/dotnet-svcutil.xmlserializer-instructions.md

```diff
@@ -0,0 +1,80 @@
+ # Using svctuil.xmlserializer on .NET Core
+
+Just like the svcutil XmlSerializer Type Generation function on desktop, dotnet-svcutil.xmlserializer NuGet package is the solution for .NET Core and .NET Standard Libraries. It pre-generates c# serialization code for the types used by Service Contract in the client applications that can be serialized using the XmlSerializer to improve the startup performance of Xml Serialization when serializing or de-serializing objects of those types using XmlSerializer. 
+
+You can start using the tool today following the instructions below. 
+
+## Prerequisites
+
+The following is required for svcutil.xmlserializer to work. You can use command `dotnet --info` to check which versions of .NET Core SDK and runtime you may already have installed.
```

Comment:
This would be easier to read if you wrote the "you can use ..." after the bulleted list.

Reply:
ok